ts/src/yobit.ts

```diff
@@ -558,19 +557,21 @@ export default class yobit extends Exchange {
         symbols = this.marketSymbols (symbols);
         let ids = undefined;
         if (symbols === undefined) {
-            const numIds = this.ids.length;
-            ids = ids.join ('-');
-            const maxLength = this.safeInteger (this.options, 'fetchTickersMaxLength', 2048);
-            // max URL length is 2048 symbols, including http schema, hostname, tld, etc...
-            if (ids.length > this.options['fetchTickersMaxLength']) {
-                throw new ArgumentsRequired (this.id + ' fetchTickers() has ' + numIds.toString () + ' markets exceeding max URL length for this endpoint (' + maxLength.toString () + ' characters), please, specify a list of symbols of interest in the first argument to fetchTickers');
-            }
+            ids = this.ids;
         } else {
-            const newIds = this.marketIds (symbols);
-            ids = newIds.join ('-');
+            ids = this.marketIds (symbols);
```

Comment:
@ttodua we have an issue here
![image](https://github.com/ccxt/ccxt/assets/43336371/69e8de00-630f-45b5-abbb-43649ee4593d)

Reply:
yes, I was in the thoughts what would be the fix for it.
Initially I thought that we could make 2-3 parallel request with promise.all, but it turns out that yobit would need 20 requests to cover all tickers. so, we shouldnt do that much in promise.all.
then, the only thing we can make it now, is to skip it in tests (or just set limited symbols amount)

Reply:
ah wait, for 7 markets it shouldn't fail. let me check php specifically, maybe transpiling issue.